client/lib/invites/reducers/invites-sent.js

```diff
@@ -1,34 +0,0 @@
-/**
- * Internal dependencies
- */
-import { action as ActionTypes } from 'calypso/lib/invites/constants';
-
-export const initialState = {
-	successes: {},
-	errors: {},
-};
-
-export const reducer = ( state = initialState, { action: { type, data, formId } } ) => {
-	switch ( type ) {
-		case ActionTypes.RECEIVE_SENDING_INVITES_ERROR:
```

Comment:
Seems like we can also delete `RECEIVE_SENDING_INVITES_SUCCESS` from `lib/invites/constants`.